### PR TITLE
General cleanup and support for object literal config

### PR DESCRIPTION
**Pull request by ded**
I also made the example point to the local reference of smoosh rather than what's installed from npm — this way changes to the index will be reflected when running the examples

### CODE REVIEW

File: lib/smoosh/index.js [hunk 1]
```diff
@@ -26,28 +26,24 @@ var SMOOOOOOOSH = '\
...
     this.newLine();
     console.log('Whoa! You fucked up... smoosh needs a valid config file!\n'.yellow);
   },
-  
+
   jshint: function(key, errors) {
     if (!errors.length) {
-
-      console.log('+' + ' Congratulations. You are a very special and Handsome person.'.green);
-      console.log('  '.green + key.yellow + ' PASSED jshint with no errors!\n');
```
Comment (fat) [inline lib/smoosh/index.js:43]: if you do mulitple packages... you wont know what file passed if you don't call that out with the key

File: lib/smoosh/index.js [hunk 2]
```diff
@@ -26,28 +26,24 @@ var SMOOOOOOOSH = '\
...
-
+      console.log('+' + ' Congratulations. You are the perfect JavaScripter.'.green);
     } else {
-
       console.log([
-        'Boo!' 
-      , '*' + key + '*', 
-      , 'FAILED jshint with'
+        'Boo!'
+      , '*' + key + '*',
```
Comment (fat) [inline lib/smoosh/index.js:45]: we have too many comas here... ;)

File: lib/smoosh/index.js (hunk 1 above)
Comment (fat) [inline lib/smoosh/index.js:43]: also, we should just switch all commas to trailing, if not comma first

File: examples/makefile.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-var smoosh = require('smoosh');
```
Comment (fat) [inline examples/makefile.js:1]: maybe leave this commented out with an example that says how they should use smoosh

```
var smoosh = require('smoosh'); //<-- use this in your projects...
```

Comment (ded) [inline examples/makefile.js:1]: yeah. in general tho there just needs to be actual tests with proper docs.

**Pull request merged**